requirements/edx/github.txt

```diff
@@ -79,7 +79,7 @@ git+https://github.com/edx/XBlock.git@xblock-0.4.13#egg=XBlock==0.4.13
 git+https://github.com/edx/edx-ora2.git@1.1.13#egg=ora2==1.1.13
 -e git+https://github.com/edx/edx-submissions.git@1.1.4#egg=edx-submissions==1.1.4
 git+https://github.com/edx/ease.git@release-2015-07-14#egg=ease==0.1.3
-git+https://github.com/edx/i18n-tools.git@v0.3.7#egg=i18n-tools==v0.3.7
+git+https://github.com/edx/i18n-tools.git@v0.3.7#egg=edx-i18n-tools==v0.3.7
```

Comment:
This installs the package under a new egg name but does not remove the old install, will that cause issues during import?  Like two packages that provide the same module?